instrumentation/opentelemetry-instrumentation-redis/src/opentelemetry/instrumentation/redis/util.py

```diff
@@ -0,0 +1,68 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+"""
+Some utils used by the redis integration
+"""
+
+from opentelemetry.instrumentation.redis import constants
+
+VALUE_PLACEHOLDER = "?"
+VALUE_MAX_LEN = 100
+VALUE_TOO_LONG_MARK = "..."
+CMD_MAX_LEN = 1000
+TARGET_HOST = "out.host"
```

Comment:
there's a bunch of constants that are re-used in many instrumentation modules, it would make sense for the sake of consistency to reuse the same constants. I only copied and pasted them in this module for convenience, but likely this will need to go to a common place sooner than later